skimage/measure/_regionprops.py

```diff
@@ -358,22 +358,73 @@ def __eq__(self, other):
         return True
 
 
+def to_dict(regions, want=['label', 'bbox'], separator='-', cancel_auto=False):
```

Comment:
This function name is too generic. I suggest calling this `_props_to_dict`, and then adding a separate function, `regionprops_table` that runs `regionprops` followed by `_props_to_dict`.

imho the signature should be `_to_dict(props_list, properties=('label', 'bbox'), index_sep='-', always_include_label=True):`

Reply:
No, I think label should always be implied, but bbox is probably not relevant if you are asking for other things.